superbench/benchmarks/micro_benchmarks/build.sh

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) Microsoft Corporation - All rights reserved
+# Licensed under the MIT License
+
+ #!/bin/bash
```

Comment:
shebang does not work if it's not at the beginning of a file, `#!` must be the first two chars

Reply:
Done.